tests/formspree_test_case.py

```diff
@@ -0,0 +1,34 @@
+
+import os
+import fakeredis
+
+from flask.ext.testing import TestCase
+import formspree
+from formspree import create_app
+from formspree import settings
+from formspree.app import DB, REDIS
+
+
+class FormspreeTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        ''' configures app for testing '''
+        formspree.app.get_redis = lambda : fakeredis.FakeStrictRedis()
```

Comment:
I think in the case that we added tests that expect an actual redis we could then implement teardown to restore the redis. I don't think we need this now, though, and I can't think of tests that would require an actual redis db.